### PR TITLE
Add French Translations

### PR DESCRIPTION
**Pull request by Dthurow**
### What
Related to #264
Add french translation json, update language translation doc, import  the french translation into the translator.js

### Why
Adds French as a language option

### Ensure the following interactions work as expected. Please test using a mobile form factor.
- [x] Tapping on a marker on the map displays information about the marker in a popup.
- [x] Tapping the "Show list of locations" button replaces the map view with a list view.
- [x] Tapping an item in the list replaces the list view with a map view, and navigates the map to the tapped item on the map.
- [x] Tapping one of the ✅'s in the legend filters items on the map and in the list.
- [x] Changing the language to spanish changes things on the page.
- [x] Clicking the Help/Info button opens and closes a menu with information.
- [x] Clicking the X Close button on the Help/Info screen closes the modal.

### Check the app in the following web browsers:
- [x] Chrome
- [x] Firefox
- [ ] Edge
- [ ] Safari

### CODE REVIEW

File: docs/LANGUAGE_TRANSLATION.md [hunk 1]
```diff
@@ -94,18 +92,7 @@ If the language translator contributed their translations using a dedicated Goog
...
-
-https://en.wikipedia.org/wiki/Unrepresented_Nations_and_Peoples_Organization#Members
-
-A checklist for adding new flag images
-
- - [ ] The image should be a PNG
- - [ ] The image should be 23px wide
- - [ ] The image name should have the format `lang-XXX.png` where "XXX" is the 3-letter language code
- - [ ] The image should be placed in the `/public/images` directory in this repo
+If the language translator contributed their translation in a Google Sheet with all other translations, you can copy the new translation into a new dedicated spreadsheet and use the above instructions to create the json file. Make sure to copy the ID column, the English column, then the column of the new language translations. This is the only way for the `/bin/google-sheet-with-translation-to-json` to work correctly.
```
Comment (amaxama) [inline docs/LANGUAGE_TRANSLATION.md:95]: Thanks for all the updates to this file, and this is a great addition!

File: src/i18n/fra.json [hunk 2]
```diff
@@ -0,0 +1,88 @@
+{
+  "locale": "fra",
```
Comment (amaxama) [inline src/i18n/fra.json:2]: Something I'm still a little fuzzy on is how all the moment locale stuff works (as you may have seen from previous PRs/reviews haha).  Is that something you looked into at all? It seems there is a `fr`  locale available so should that be used here?

Comment (Dthurow) [reply src/i18n/fra.json:2]: Well I thought I understood it till I was double checking it. I was just putting in the 3 letter code from the linked wikipedia article in the Language Translation document, and the places I was seeing relative time things (e.g. "last updated 3 days ago") was translating, so I thought I did that correctly. But you're right, moment.js lists the locale code as fr, not fra. And when I double checked that just now, it's leaving it translated in the wrong language, even when I change the locale to "fr". So I'll need to poke around more. Good catch!

Comment (Dthurow) [reply src/i18n/fra.json:2]: ah ha! So moment can let you download all their default locales at once using the standard npm install moment, but that takes up a lot of space, so in #111 , it was updated so we'd only pull in the default momentjs locales that we were currently using, and our custom locales. 
We can get copies of the momentjs locales on their github: https://github.com/moment/moment/tree/develop/locale, and copy them int the web_modules/moment/dist folder, then load them into the index.js file. Once I did that, the moment.js translations for french work (also after I updated the fra.json locale to "fr"). 
I'll do a new push with those changes and updating the translation doc with this info now.

Comment (amaxama) [reply src/i18n/fra.json]: Fantastic! Great find! I think this was one of the last pieces of moment that wasn't quite clicking in our set up so thanks for digging into that!

**Review by amaxama — APPROVED** [review]

**Pull request merged**